### DISCUSSION TITLE
command about hide balance stats at baltop

### DISCUSSION THREAD

**Discussion by MusicTW** [Zrips/CMI#8127] [Enhancement]
My players just told me that they want to hide their stats at baltop
And I found that I can only do add their name to config, to hide their bal stats at baltop
And I think if there's a way like do '/cmi balance %player% hide' to let player can hide their stats at baltop

**Comment by CloudeLecaw** [comment]
What abbount adding the command with a special needed permission? Might not everyone whats to allow his users to hide from balancetop.
The command might could be simply add or remove a player from the configlist to just stay with one points where hidden baltop players are stored.

 if you condider about adding this `cmi balance (playername) (hidetop|unhidetop)` pls also add the permission `cmi.command.balance.hidetop`  for this command. so we can control who is allowed to hide from balancetop :3

**Reply by CloudeLecaw** [reply]
at the moment we use a plugin like rpgbanker where players can dispose the money. it likits their balance when its deposite in the bank. and in result they wont got counted for richest players ;)
